### PR TITLE
Preserve TextureUsage when caching on the CPU if it was previously set to PACK

### PR DESCRIPTION
**Pull request by annxingyuan**
This PR changes `cacheOnCPU` so that the texture usage is preserved if it was previously `TextureUsage.PACK`. 

This will be useful for implementing lazy packing / unpacking as part of packing batchNormalization (https://github.com/tensorflow/tfjs/issues/774). When `DEBUG` is on, we call `cacheOnCPU` for every texture in order to check for NaN's. This sets the `TextureUsage` of the texture to `UPLOAD`, regardless of its original `TextureUsage`. This makes it difficult to lazily unpack while debugging. 

BUG

---

##### For repository owners only:

Please remember to apply all applicable tags to your pull request.
Tags: FEATURE, BREAKING, BUG, PERF, DEV, DOC, SECURITY

For more info see: https://github.com/tensorflow/tfjs/blob/master/DEVELOPMENT.md

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-core/1328)

### CODE REVIEW

**Comment by annxingyuan** [comment]
This is no longer an issue since https://github.com/tensorflow/tfjs-core/pull/1329 separated `usage` from `isPacked`
